java/doc/api.rst

```diff
@@ -114,129 +108,47 @@ as the type of an input parameter.
 ``RayList<T>``
 ^^^^^^^^^^^^^^
 
-This is a list of ``RayObject<T>``s, which inherits from ``List<T>`` in Java. It
+This is a list of ``RayObject<T>``'s, which inherits from ``List<T>`` in Java. It
```

Comment:
I think there should not be an `'` here. I wasn't completely sure, but this is my takeaway from http://grammar.ccc.commnet.edu/grammar/plurals.htm.

From the link:

*We use an apostrophe to create plural forms in two limited situations:*
- *for pluralized letters of the alphabet*
- *when we are trying to create the plural form of a word that refers to the word itself*

Reply:
👍TIL.
I saw you changed one `RayObject` to "`RayObject`'s" in [this commit](https://github.com/ray-project/ray/pull/2491/commits/1abe6d11de5a6b20d29c92bcf7a35d01e255b1eb). Is that case different? Should I change it back?

Reply:
Whoops, you're right, I think it should be changed back.